charts/stable/wordpress/templates/_env.tpl

```diff
@@ -1,6 +1,6 @@
 {{/* Wordpress environment variables */}}
 {{- define "wordpress.env" -}}
-  {{- $configName := printf "%s-env-config" (include "tc.common.names.fullname" .) }}

```

Comment:
??

Reply:
i didnt remove that, my end still has it.

Reply:
oh, i see what happen